warmup/summary_statistics.py

```diff
@@ -357,16 +354,15 @@ def write_latex_table(machine, all_benchs, summary, tex_file, num_splits,
                 fp.write('\\begin{table*}[hptb]\n')
                 fp.write('\\vspace{.8cm}\n')
                 fp.write('\\begin{adjustbox}{totalheight=12.4cm}\n')
-        # emit table header
-        heads1 = TABLE_HEADINGS_START1 + '&'.join([TABLE_HEADINGS1] * num_splits)
-        heads2 = TABLE_HEADINGS_START2 + '&'.join([TABLE_HEADINGS2] * num_splits)
+        heads1 = TABLE_HEADINGS_START1 + '&'.join([TABLE_HEADINGS1])
+        heads2 = TABLE_HEADINGS_START2 + '&'.join([TABLE_HEADINGS2])
         heads = '%s\\\\%s' % (heads1, heads2)
         if longtable:
             fp.write(start_longtable(TABLE_FORMAT, heads))
         else:
             fp.write(start_table(TABLE_FORMAT, heads))
         split_row_idx = 0
-        for row_vms in zip(*splits):
+        for row_vms in zip(*[all_vms]):
```

Comment:
Is `all_vms` a list or a list of lists?

Reply:
It's a list. This was the source of all of my confusion this week! BTW this line was changed in a later commit.